app/controllers/api/v1/web_hooks_controller.rb

```diff
@@ -30,14 +30,24 @@ def remove
   end
 
   def fire
-    webhook = current_user.web_hooks.new(url: @url)
-    @rubygem = Rubygem.find_by_name("gemcutter") unless @rubygem
+    rubygem = @rubygem ? @rubygem : Rubygem.find_by_name("gemcutter")
```

Comment:
@shlok007 can you please change it to what @esparta suggsted? It is more readable.